src/navi/core.clj

```diff
@@ -158,8 +168,60 @@
              body-spec
              [:or nil? body-spec])}))
 
+;;; Handle reponses
+
+(defn linked-hash-map->clj-map
+  "Convert a Java LinkedHashMap to a Clojure map.
+  Preserve nils."
+  [key-fn val-fn ^LinkedHashMap hm]
+  (if (nil? hm)
+    nil
+    (into {}
+          (map (fn lhm-mapper [map-entry]
+                 (vector (key-fn (.getKey ^Map$Entry map-entry))
+                         (val-fn (.getValue ^Map$Entry map-entry))))
+               hm))))
+
+(defn handle-response-key
+  "Reitit seems to want status codes of a response to be integer keys,
+  rather than keyword keys - except for :default.
+  So, convert a string to a Long if relevant.
+  If the string is \"default\" then return it as a keyword, otherwise pass through.
+  Arguably, all non-integer status codes should be converted to keywords."
+  [s]
```

Comment:
is it possible that the user supplies an int via the yaml/json? would this regex match still work?

Reply:
The [OpenAPI Spec](https://spec.openapis.org/oas/v3.1.0.html#patterned-fields-0) specifically states that the response code should be a string.  And, the `ApiResponse` class is declared to be a map from `String` to `ApiResponse`
```
public class ApiResponses extends LinkedHashMap<String, ApiResponse>
```
I tried to `.put` an int as the key and I got an Exception.

So, I'm presuming that we will never get an int here, and only get a String.

Reply:
Makes sense. Thanks for verifying!